code/io/joy_ff-sdl.cpp

```diff
@@ -108,13 +114,29 @@ int joy_ff_init()
 {
 	bool ff_enabled;
 
+	if (Joy_ff_enabled) {
+		joy_ff_shutdown();
+	}
```

Comment:
Most of the other startup functions in the codebase (e.g. `ship_init()`) will just return if something is already initialized, leaving it initialized.  That's what I'd recommend here, unless there's a reason to "reboot" the force-feedback.

Reply:
Yeah, an easy re-init was the intention, and wasn't put in for use as a guard or anything. Although the final iteration of these changes no longer depends on that functionality, it was left in more for possible future use.

It may be better converted to a guard though and then adding a simple `joy_ff_reinit()` for the specific purpose of re-initializing the ff code. It would go unused in the current code but would probably be a cleaner and more obvious way to go about it.